### PR TITLE
Add text to verification section

### PR DESCRIPTION
**Pull request by brentzundel**
This PR adds some text to the Verification section, adds some text to the content integrity section, adds text to the verifiable data registry subsection of the CIP section, and makes the hashlinks section a subsection of the CIP section. (moving the hashlinks section and creating the CIP subsection were part of previous, already merged PRs, but were lost due to rash conflict resolution)
Signed-off-by: Brent <brent.zundel@gmail.com>

***
<a href="https://pr-preview.s3.amazonaws.com/brentzundel/vc-imp-guide/pull/40.html" title="Last updated on Aug 15, 2019, 3:33 PM UTC (e4f583c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-imp-guide/40/757905a...brentzundel:e4f583c.html" title="Last updated on Aug 15, 2019, 3:33 PM UTC (e4f583c)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -187,7 +187,12 @@ <h2>Verification</h2>
       <em>This section is non-normative.</em>
 
       <p>
-TBD
+<a>Verification</a> is the process a <a>verifier</a> or <a>holder</a> undergoes
```
Comment (TallTed) [inline index.html]: `undergoes`
-> `performs`
(The VC undergoes this process.)

File: index.html [hunk 2]
```diff
@@ -207,6 +212,35 @@ <h3>Specific Verifiable Credentials</h3>
...
+        <h3>Content Integrity</h3>
+        <p>
+Protecting the integrity of content is an important component of verification.
+<a>Verifiers</a> need to have confidence that the content they rely on to
+verify <a>credentials</a> doesn't change without their knowledge. This content
+may include data schemas, identifiers, public keys, etc.
+        </p>
+        <p>
+There are a number of ways to provide content integrity protection. A few of
+them are provided in greater detail below.
```
Comment (TallTed) [inline index.html]: `them are provided`
-> `these are described`

Comment (brentzundel) [reply index.html]: fixed in [058b883](https://github.com/w3c/vc-imp-guide/pull/40/commits/058b883d0adafa4a54c8c49fddce42146f84ba31)

File: index.html (hunk 1 above)
Comment (brentzundel) [reply index.html]: fixed in [058b883](https://github.com/w3c/vc-imp-guide/pull/40/commits/058b883d0adafa4a54c8c49fddce42146f84ba31)

File: index.html (hunk 2 above)
Comment (TallTed) [reply index.html]: `them` remains ... but I'll live with it

Comment (brentzundel) [reply index.html]: and [9faa5c7](https://github.com/w3c/vc-imp-guide/pull/40/commits/9faa5c74747744aac2da8c1c9ab646132ba69118)

File: index.html [hunk 3]
```diff
@@ -207,6 +212,35 @@ <h3>Specific Verifiable Credentials</h3>
...
+        </p>
+        <p>
+There are a number of ways to provide content integrity protection. A few of
+these are described in greater detail below.
+        </p>
+        <section>
+          <h4>Hashlinks</h4>
+
+          <p>
+Hashlink URLs can be used to provide content integrity for links to external
```
Comment (deiu) [inline index.html]: We should add a pointer to https://tools.ietf.org/html/draft-sporny-hashlink-00 or some other resource that describes in detail what hashlinks are.

File: index.html [hunk 4]
```diff
@@ -207,6 +212,35 @@ <h3>Specific Verifiable Credentials</h3>
...
+resources.
+          </p>
+        </section>
+        <section>
+          <h4>Verifiable Data Registries</h4>
+          <p>
+A <a>verifiable data registry</a> can also provide content integrity protection.
+One example of a <a>verifiable data registry</a> which provides content
+integrity protection is a distributed ledger.
```
Comment (deiu) [inline index.html:240]: I'm not a big fan of adding links to Wikipedia in specs, but we need a reference for distributed ledgers here too. (https://en.wikipedia.org/wiki/Distributed_ledger)

**Review by deiu — CHANGES_REQUESTED** [review]
Looks good, with the exception of just a couple of missing references.

File: index.html (hunk 3 above)
Comment (brentzundel) [reply index.html]: I will add the link, and agree that the hashlink section needs more detail. I don't feel qualified to add that detail, so I will raise a separate issue for that.

File: index.html (hunk 4 above)
Comment (brentzundel) [reply index.html:240]: I am going to add quite a bit more detail here. I will raise an issue to track that work so that this PR isn't hung up on it.

**Comment by brentzundel** [comment]
The primary goal of this PR is to organize the Verification section and add a bit of starter text. I hope others will add more PRs to better flesh it out.

File: index.html (hunk 3 above)
Comment (brentzundel) [reply index.html]: I've added a comment to Issue #31 to track this

File: index.html (hunk 4 above)
Comment (brentzundel) [reply index.html:240]: I've added a comment to Issue #31 to track this

Comment (deiu) [reply index.html:240]: 👍

**Review by deiu — APPROVED** [review]

**Comment by brentzundel** [comment]
I added the reference to hashlinks, but did not add the wikipedia link for distributed ledgers because I plan to extend that section and will hopefully cover what the link would.

**Pull request merged**